galette/templates/default/pages/preferences.html.twig

```diff
@@ -22,7 +22,7 @@
 
 {% block content %}
         <form action="{{ url_for('store-preferences') }}" method="post" enctype="multipart/form-data" class="ui form">
-        <div class="ui stackable pointing inverted menu tabbed">
+        <div class="ui stackable pointing inverted wrapping menu tabbed">
```

Comment:
Waht about?
```suggestion
        <div class="ui stackable pointing inverted wrapping centered fluid menu tabbed">
```

Reply:
I've applied since I prefer this display (and all seems OK) :)

Reply:
The "drawback" of this variation is that it applies also on large screens, making tabs wider.

Default:
![large-default-preferences](https://github.com/galette/galette/assets/107203963/58f31e02-65ef-4f28-b7bc-c60d56e0afd7)

Centered fluid:
![large-centered-fluid-preferences](https://github.com/galette/galette/assets/107203963/9d8f265c-8453-4f5f-a1c9-487923b860e5)

For UI consistancy, shouldn't we use it on every screen ?
But with few tabs only, I personnaly don't find that great and I'd prefer the tabs to remain "compact".

Default:
![large-default-few-tabs](https://github.com/galette/galette/assets/107203963/b4b24396-6973-4745-a106-86a52a33f3ab)

Centered fluid:
![large-centered-fluid-few-tabs](https://github.com/galette/galette/assets/107203963/4e06f70f-e24f-4c27-87f7-cc34e62a703c)

It's up to you ! :wink:

And I suppose that if it is used on preferences only, it won't break the UI consistancy that much :grin:

Reply:
> The "drawback" of this variation is that it applies also on large screens, making tabs wider.
> [...]
> It's up to you ! 😉
> 
> And I suppose that if it is used on preferences only, it won't break the UI consistancy that much 😁

Indee I did no think about other tabs usage; but preferences is the only one that needs wrapping; and in that cas the fluid fits correctly. I'll keep it; it's easy to change anyway :smile: